com.unity.visualeffectgraph/Editor/Inspector/VFXSlotContainerEditor.cs

```diff
@@ -301,8 +301,10 @@ static Styles()
             warningStyle.margin.right = 1;
 
             frameButtonStyle = new GUIStyle();
-            frameButtonStyle.normal.background = EditorGUIUtility.Load(EditorResources.iconsPath + "d_ViewToolZoom.png") as Texture2D;
-            frameButtonStyle.active.background = EditorGUIUtility.Load(EditorResources.iconsPath + "d_ViewToolZoom On.png") as Texture2D;
+            frameButtonStyle.normal.background = EditorGUIUtility.LoadIconForSkin(EditorResources.iconsPath + "ViewToolZoom.png", EditorGUIUtility.skinIndex);
+            frameButtonStyle.active.background = EditorGUIUtility.LoadIconForSkin(EditorResources.iconsPath + "ViewToolZoom On.png", EditorGUIUtility.skinIndex);
+            frameButtonStyle.normal.background.filterMode = FilterMode.Trilinear;
+            frameButtonStyle.active.background.filterMode = FilterMode.Trilinear;
```

Comment:
I don't know if the icon texture have mipmaps, if they don't, then FilterMode.Bilinear is enough. Anyways, better safe then sorry, so Trilinear is fine to me :)

Reply:
Yep, for two icons I'm not worried about the cost of trilinear filtering and feel safe to the visual quality.